source/Databricks/source/Jobs/Diagnostics/HealthChecks/DatabricksJobsApiHealthRegistration.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright 2020 Energinet DataHub A/S
+//
+// Licensed under the Apache License, Version 2.0 (the "License2");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Energinet.DataHub.Core.Databricks.Jobs.AppSettings;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using NodaTime;
+
+namespace Energinet.DataHub.Core.Databricks.Jobs.Diagnostics.HealthChecks;
+
+public class DatabricksJobsApiHealthRegistration : IHealthCheck
```

Comment:
This is not a registration, this is the health check, and should be named accordingly.
```suggestion
public class DatabricksJobsApiHealthCheck : IHealthCheck
```